### DISCUSSION TITLE
Is it possible to add an image to email template?

### DISCUSSION THREAD

**Discussion by Ryandonofrio3** [supabase/supabase#13248] [Questions]
Trying to make my first email from Supabase as a sign up confirmation. 

`<table style="width:100%;max-width:600px;margin:0 auto;">
  <tbody><tr>
    <td style="padding:10px;">
      <div style="display:flex;align-items:center;justify-content:center;">
        <img style="height:50px;margin-right:10px;" src="xxxxx" alt="xxxxx Logo">
      </div>
    </td>
  </tr>
  <tr>
    <td style="padding:10px;">
      <div style="display:flex;flex-direction:column;align-items:center;text-align:center;font-family:'Raleway', sans-serif;">
        <h3 style="margin-bottom:0;">Hello,</h3>
        <h3 style="margin-top:0;margin-bottom:10px;">You have requested a magic link to automatically sign in to xxxxxx</h3>
        <p style="margin-bottom:10px;">
          <a style="display:inline-block;background-color:#991b1b;color:#fff;padding:10px 20px;border-radius:25px;text-decoration:none;font-weight:bold;margin-top:10px;border:2px solid #991b1b;font-family:'Raleway', sans-serif;" href="{{ .ConfirmationURL }}">Click here to sign in</a>
        </p>
        <p style="margin-bottom:10px;">Or copy-paste the following link into your browser:</p>
        <p style="margin-bottom:10px;">{{ .ConfirmationURL }}</p>
        <h3 style="margin-top:0;margin-bottom:0;">Thank you!</h3>
        <h3 style="margin-top:0;"> xxxxxxxx Team</h3>
      </div>
    </td>
  </tr>
</tbody></table>`

but it seems to get rid of my image. any way around this? same with the button!

Thanks!

**Comment by GaryAustin1** [comment]
Embedded images should work.
```
<center>
<img src="https://xxxxx.supabase.co/storage/v1/object/public/public/IMG_1966.JPG">
</center>
```
Works for me.

**Reply by KristiyanTs** [reply]
Now we get a warning:
"Email has a high probability of being marked as spam - review issues below to improve deliverability."
"URI hostname has a long non-vowel sequence"

This is because the supabase storage has a strange url: _https://wnatkawbbzsncoquhohe.supabase.co/storage/v1/object/public/static//logo.svg_

**Reply by GCheung55** [reply]
> Now we get a warning: "Email has a high probability of being marked as spam - review issues below to improve deliverability." "URI hostname has a long non-vowel sequence"
> 
> This is because the supabase storage has a strange url: _https://wnatkawbbzsncoquhohe.supabase.co/storage/v1/object/public/static//logo.svg_

I'm also seeing this issue. The storage URL has a long subdomain that seems to be the cause of the warning.

**Reply by hudahoeda** [reply]
I also get that issue when using URL from supabase bucket, if we want to get rid of it one of the possible solution is using the custom domain for the supabase project, but might worth trying the URL from supabase bucket, if it marked as spam you should use the custom domain
